File: Python/Warmup-2/string_bits.py
"""
Given a string, return a new string made of every other char starting with the first, so "Hello" yields "Hlo".


string_bits('Hello') → 'Hlo'
string_bits('Hi') → 'H'
string_bits('Heeololeo') → 'Hello'
"""

def string_bits(str):
  str1=""
  for i in range(0,len(str),2):
    str1=str1+str[i]
  return str1

File: Java/Warmup-2/stringBits.java
/*

Given a string, return a new string made of every other char starting with the first, so "Hello" yields "Hlo".


stringBits("Hello") ? "Hlo"
stringBits("Hi") ? "H"
stringBits("Heeololeo") ? "Hello"
*/

public String stringBits(String str) {
  String str1="";
  for(int i=0;i<str.length();i=i+2){
    str1=str1+str.charAt(i);
  }
  return str1;
}
